app/lib/server/startup/settings.js

```diff
@@ -61,6 +61,10 @@ settings.addGroup('Accounts', function() {
 		type: 'boolean',
 		public: true,
 	});
+	this.add('Accounts_AllowEphemeralChannels', true, {
```

Comment:
Remember to add an english translation 🙏

Reply:
I have added already!